integration/single/test_basic_layer_version.py

```diff
@@ -1,6 +1,10 @@
+from unittest.case import skipIf
+
 from integration.helpers.base_test import BaseTest
+from integration.helpers.resource import should_exclude_test_in_region
 
 
+@skipIf(should_exclude_test_in_region("Layers"), "Layers is not supported in this testing region")
```

Comment:
I think it is a good idea to move the skip if decorator to the testing methods, so any new test case will be added to this class will not be skipped by default, and if it is needed to be skipped, the developer should explicitly add the skip if decorator

Reply:
Agree, will update.